.lintr

```diff
@@ -1,12 +1,12 @@
 linters: with_defaults(
     line_length_linter = line_length_linter(80),
-    cyclocomp_linter = cyclocomp_linter(complexity_limit = 25L),
+    cyclocomp_linter = cyclocomp_linter(complexity_limit = 50L),
```

Comment:
eek

Reply:
Yeah pretty much all for the test, actual is ~32. Theres nothing new though so I imagine this would have shown up if we didn't exclude the tests from linting.